### DISCUSSION TITLE
Cannot create new Environment for new Application in Noop Workshop 0.4.1

### DISCUSSION THREAD

**Discussion by nprimmer** [noop-inc/documentation#88] [Bugs] [answered]
When creating a new Application in Noop Workshop 0.4.1, I am unable to create a new environment. Screenshots of configuration selected and the error received are attached below.

<img width="807" alt="Screenshot 2024-01-24 at 10 16 09 AM" src="https://github.com/noop-inc/documentation/assets/2281777/e063bf4b-fbc9-40a1-8385-bcbcd0ff979a">
<img width="799" alt="Screenshot 2024-01-24 at 10 16 18 AM" src="https://github.com/noop-inc/documentation/assets/2281777/fe351578-3827-41ae-aafa-09760670acec">

**Comment by derekwolpert** [comment, marked as answer]
Thanks for the heads up here @nprimmer. A fix for this issue will be included in the next release of Noop Desktop.
